ca/ca.go

```diff
@@ -227,9 +227,11 @@ func (ca *CA) Reload() error {
 	}
 
 	// 1. Stop previous renewer
-	// 2. Replace ca properties
+	// 2. Close key manager
```

Comment:
Change this line to something like: `Safely shutdown any internal resources (e.g. key manager)`